server/src/coqtop/CoqTop.ts

```diff
@@ -102,7 +102,7 @@ export function detectVersion(coqtopModule: string, cwd: string, console?: {log:
       });
 
       coqtop.on('close', () => {
-        const ver = /^\s*The Coq Proof Assistant, version (.+?)\s/.exec(result);
+        const ver = /^\s*The (?:Coq Proof Assistant|Rocq Prover), version (.+?)\s/.exec(result);
```

Comment:
maybe it would be better to call `coqtop --print-version` whose result should be easier to parse (it's output of `printf "%s %s" coq_version ocaml_version`)
--print-version is available since coq 8.6 AFAICT

Reply:
I think it is ok like this, hope we will not change name again anytime soon :smiley: